Gopkg.toml

```diff
@@ -49,6 +50,10 @@ ignored = [
   name = "github.com/stretchr/testify"
   version = "1.2.1"
 
+[[constraint]]
+  name = "github.com/go-kit/kit"
+  version = "0.6.0"
+
```

Comment:
I'm pretty sure that this constraint will be ignored by `dep` because the whole package has been ignored, if you run `dep ensure -v` it'll tell you for sure.

Reply:
Yeah. Removed it.